### ISSUE TITLE
Get discounts from local hotels in Bled 

### ISSUE THREAD

**Issue by yangwao** [kodadot/subwork#14]
I would start from local radius and top rating, here is your headstart 

- https://www.google.com/maps/search/Hotels/@46.3656193,14.0605843,13z/data=!3m1!4b1!4m4!2m3!5m1!4e9!6e3

**Comment by yangwao** [comment]
@helloitsdamsky can we drop number for GAIA? we can deal with them for some discount if we are sustainable and would drive traffic to them 😄

whenever ppl refer to subwork

**Comment by yangwao** [comment]
https://maps.app.goo.gl/qZEfDAakwU6E9Vt87

+386 64 207 778

is there any other numero?

**Comment by helloitsdamsky** [comment]
> https://maps.app.goo.gl/qZEfDAakwU6E9Vt87
> 
> +386 64 207 778
> 
> is there any other numero?

No, that one is to be used. However i always say we have Subwork co-working in Trgovica and we are coming often. I can jump there once i’m on bled and try to set-up some discount or "code"

**Comment by yangwao** [comment]
added Gaja Apartments

f7daff1..7c69a7e

**Comment by yangwao** [comment]
This will be fantastic for SubWork Fellowship
- #156 
- https://subwork.xyz/subwork-fellowship
